src/models/ImgixModel.php

```diff
@@ -297,7 +297,7 @@ public function srcset($attributes = []): ?Markup
             foreach ($images as $image) {
                 $width = $image['width'] ?? $image['w'] ?? null;
                 if ($width && !isset($widths[ $width ])) {
-                    $withs[ $width ] = true;
+                    $widths[ $width ] = true;
```

Comment:
I noticed a typo on the variable here. 

Hopefully this doesn't cause an unintended change.